APIHook.cpp

```diff
@@ -119,7 +119,7 @@ class ChronosFileOpenDialog : public IFileOpenDialog
 		}
 
 		CComPtr<IShellItem> psi;
-		hresult = ::SHCreateShellItem(NULL, NULL, pidl, &psi);
+		hresult = ::SHCreateItemFromIDList(pidl, IID_PPV_ARGS(&psi));

```

Comment:
REFIID is not specified, does it correct? (SHCreateItemFromIDList got 2 arguments, but it requires 3 arguments, I thought)

Reply:
got it.`IID_PPV_ARGS` expands them.

Reply:
yes, `IID_PPV_ARGS` expands them.

https://learn.microsoft.com/ja-jp/windows/win32/api/combaseapi/nf-combaseapi-iid_ppv_args

```
#define IID_PPV_ARGS(ppType) __uuidof(**(ppType)), IID_PPV_ARGS_Helper(ppType)
```